modules/load-balancer-target-group/main.tf

```diff
@@ -43,13 +52,43 @@ resource "aws_alb_listener_rule" "http_path" {
   priority     = var.listener_rule_starting_priority + count.index
 
   action {
-    target_group_arn = aws_alb_target_group.tg.arn
     type             = "forward"
+    target_group_arn = aws_alb_target_group.tg.arn
   }
 
-  condition {
-    field  = var.routing_condition.field
-    values = var.routing_condition.values
+  dynamic "condition" {
```

Comment:
Use the same approach as couchbase: https://github.com/gruntwork-io/terraform-aws-couchbase/pull/64

Reply:
Decided not to do that as this one is using a single map for the condition, as opposed to multiple in a list.